Resources/Prototypes/_NF/Catalog/pirate_uplink_catalog.yml

```diff
@@ -0,0 +1,236 @@
+      whitelist:
+        - PirateCaptain
+
+- type: listing
+  id: UplinkPirateCrateFilled
+  name:  uplink-pirate-crate-name
+  description: uplink-pirate-crate-desc
+  productEntity: CratePirateChest
+  icon: { sprite: _NF/Structures/Storage/Crates/piratechestgrey.rsi, state: crate_icon }
+  cost:
+    Doubloon: 8
+  categories:
+    - UplinkPirateUtility
+  conditions:
+    - !type:StoreWhitelistCondition
+      whitelist:
+        tags:
+          - PirateUplink
+
+- type: listing
+  id: UplinkPirateCrateFilledAmmo
+  name:  uplink-pirate-crateammo-name
+  description: uplink-pirate-crateammo-desc
+  productEntity: CratePirateChestAmmo
+  icon: { sprite: _NF/Structures/Storage/Crates/piratechestgrey.rsi, state: crate_icon }
+  cost:
+    Doubloon: 5
+  categories:
+    - UplinkPirateUtility
+  conditions:
+    - !type:StoreWhitelistCondition
+      whitelist:
+        tags:
+          - PirateUplink
+
+- type: listing
+  id: UplinkPirateSatchelFilled
+  name:  uplink-pirate-satchel-name
+  description: uplink-pirate-satchel-desc
+  productEntity: ClothingBackpackPirateFilled
+  icon: { sprite: Clothing/Back/Satchels/leather.rsi, state: icon }
+  cost:
+    Doubloon: 6
+  categories:
+    - UplinkPirateUtility
+  conditions:
+    - !type:StoreWhitelistCondition
+      whitelist:
+        tags:
+          - PirateUplink
+
+- type: listing
+  id: UplinkPirateMagboots
+  name:  uplink-pirate-magboots-name
+  description: uplink-pirate-magboots-desc
+  productEntity: ClothingShoesBootsMagPirate
+  icon: { sprite: _NF/Clothing/Shoes/Boots/magboots-pirate.rsi, state: icon }
+  cost:
+    Doubloon: 2
+  categories:
+    - UplinkPirateUtility
+  conditions:
+    - !type:StoreWhitelistCondition
+      whitelist:
+        tags:
+          - PirateUplink
+
+- type: listing
+  id: UplinkPirateHoverbike
+  name:  uplink-pirate-hoverbike-name
+  description: uplink-pirate-hoverbike-desc
+  productEntity: VehicleHoverbikePirateKey
+  icon: { sprite: _NF/Objects/Vehicles/hoverbike.rsi, state: keys }
+  cost:
+    Doubloon: 10
+  categories:
+    - UplinkPirateUtility
+  conditions:
+    - !type:StoreWhitelistCondition
+      whitelist:
+        tags:
+          - PirateUplink
+    - !type:BuyerJobCondition
+      whitelist:
+        - PirateCaptain
+
+- type: listing
+  id: UplinkPirateBlunderbuss
+  name:  uplink-pirate-blunderbuss-name
+  description: uplink-pirate-blunderbuss-desc
+  productEntity: WeaponShotgunBlunderbuss
+  icon: { sprite: Objects/Weapons/Guns/Shotguns/blunderbuss.rsi, state: icon }
+  cost:
+    Doubloon: 4
+  categories:
+    - UplinkPirateWeapons
+  conditions:
+    - !type:StoreWhitelistCondition
+      whitelist:
+        tags:
+          - PirateUplink
+
+- type: listing
+  id: UplinkPirateRevolver
+  name:  uplink-pirate-revolver-name
+  description: uplink-pirate-revolver-desc
+  productEntity: WeaponRevolverPirate
+  icon: { sprite: Objects/Weapons/Guns/Revolvers/pirate_revolver.rsi, state: icon }
+  cost:
+    Doubloon: 3
```

Comment:
also change to 1 since only has 5 shots and fires slower than a regular revolver.